src/openms/source/ANALYSIS/OPENSWATH/TransitionTSVFile.cpp

```diff
@@ -935,6 +939,11 @@ namespace OpenMS
     rm_trans.setDetectingTransition(tr_it->detecting_transition);
     rm_trans.setIdentifyingTransition(tr_it->identifying_transition);
     rm_trans.setQuantifyingTransition(tr_it->quantifying_transition);
+
+    if (!tr_it->peptidoforms.empty())
+    {
+      rm_trans.setMetaValue("Peptidoforms", ListUtils::concatenate(tr_it->peptidoforms, "|"));
```

Comment:
why use `|` here as a separator and in the SQL use `;` ?

Reply:
I thought about using ``;`` for TSV files, as it will then be compatible with the other concatenated columns. Maybe we should use ``|`` for TraML and SQLite and ``;`` for TSV?